.github/workflows/publish_api.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       IMAGE_NAME: ${{ github.repository }}/api
 
     steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
```

Comment:
While this is only necessary for self hosted runners, I decided to keep it as it doesn't hurt, and we'll probably need it once we do figure out how to switch these actions to self hosted runners